src/serializer/ResidualFunctions.js

```diff
@@ -120,15 +160,17 @@ export class ResidualFunctions {
 
   // Generate a shared function for accessing captured scope bindings.
   // TODO: skip generating this function if the captured scope is not shared by multiple residual funcitons.
-  _createCaptureScopeAccessFunction() {
+  _createCaptureScopeAccessFunction(functionValue: FunctionValue | null) {
```

Comment:
Instead of creating separate __get_scope_binding() function inside each additional function, I would suggest we share them. Single __get_scope_binding function should be enough.